src/theme/global.scss

```
@@ -589,3 +589,38 @@ input[type='number'] {
     font-size: $fontXLarge;
   }
 }
+
+@media print {
+  /* Utility */
+  .hideInPrint {
+    display: none !important;
+  }
+
+  @page {
+    size: A4;
```

Stylelint (17.3.0)

[error] Expected "A4" to be "a4" (value-keyword-case)

(value-keyword-case)